Demo/ViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class CollectionCell: UICollectionViewCell {
```

Comment:
Yeah, I was thinking about it. Despite being just a very simplified demo. I want to make this demo better, and I thought: mhm, that could be a great place to do the Unsplash thing! What do you think? cc @3lvis